atd-toolbox/data_model/python/sql_templates/crashes_list_view.sql

```diff
@@ -241,3 +241,70 @@ left join
 left join
     lookups.injry_sev_lkp
     on lookups.injry_sev_lkp.id = crash_injury_metrics_view.crash_injry_sev_id;
+
+
+create materialized view locations_list_view as (
+    with crash_totals as (
+        with unioned_crash_counts as (
```

Comment:
i really like the way this is composed - it makes it really straightforward to see the way the data from cr3 and blueform crashes come together in this view - especially with the `0`s for the fatality and injury counts for the blueform crashes described below 🚀

not related to the changes in this PR, i did have a hard time remembering that the **Total Crashes (Previous 5 years)** row in the Details card includes both cr3 and non-cr3 counts so i initially thought there was a discrepancy between the tables at the bottom and the location 5 year total. I think that i'm just rusty, though 🙃

Reply:
Yeah that total crashes thing mike pointed out threw me for a loop as well. I was looking at a location that had 62 **Total Crashes (Previous 5 years)** but only 24 crashes in the table at the bottom. Feel like that could be acknowledged somewhere, maybe with a tooltip?

Reply:
funny you both mentioned this—Xavier also flagged that during our discussion yesterday. i agree it's confusing

Reply:
agreed—and as i'm cross-checking in prod 'm not feeling super confident about the way this is being calculated. i will spin out a follow-up issue to take a closer look at this and find out from stakeholders what behavior we actually want.